src/ui/theme/components/button.ts

```diff
@@ -40,38 +57,82 @@ function variantSolid(props: any) {
     case 'sepia':
       return {
         ...common,
+        ...commonSettingButtonStyle,
         bg: 'rgb(246, 236, 217)',
         color: 'rgb(17, 17, 17)',
+        _focus: {
+          bg: 'rgb(246, 236, 217)',
+          color: 'rgb(17, 17, 17)',
```

Comment:
These magic color values should all be added to the theme under `colors.ui` with some names

Reply:
If the values don't make sense in the theme, then they should be declared at the top of the file so they can be reused and edited easily: `const activeGrey = "rgb(...)";`